README.md

```diff
@@ -1,116 +1,59 @@
 # manta-picker
 
-This repo contains an under-development, early prototype of a standalone service that implements functionality similar to the mpicker tool in manta-muskie.  A draft RFD describing the motivation, requirements and approach for manta-picker is [here](https://github.com/joyent/rfd/tree/master/rfd/0170)
-
+The manta-picker is a Manta V2 service that provides an interface for retrieving a cached view  of the contents of the ```manta_storage``` bucket via the /poll REST endpoint.  The Rebalancer is the only current consumer of manta-picker.  Future versions of buckets-api will be modified to consume the manta-picker service.
```

Comment:
nit: do we need triple-backticks for inline markdown? I'd have thought single-backticks would be enough given the text is all inline with no carriage returns.

Reply:
You're right - fixed.